index.bs

```diff
@@ -1453,10 +1453,15 @@ URLs, is as follows:
 # Security Considerations # {#security-considerations}
 
 The Sanitizer API is intended to prevent DOM-based Cross-Site Scripting
```

Comment:
In the heading below we just use "XSS". We should probably be somewhat consistent in this document about that. And we might also need a reference for this term I suppose if we want to use it.